lib/membrane/srtcp/decryptor.ex

```diff
@@ -46,8 +46,22 @@ if Code.ensure_loaded?(ExLibSRTP) do
 
     @impl true
     def handle_process(:input, buffer, _ctx, state) do
-      {:ok, payload} = ExLibSRTP.unprotect_rtcp(state.srtp, buffer.payload)
-      {{:ok, buffer: {:output, %Buffer{buffer | payload: payload}}}, state}
+      case ExLibSRTP.unprotect_rtcp(state.srtp, buffer.payload) do
+        {:ok, payload} ->
+          {{:ok, buffer: {:output, %Buffer{buffer | payload: payload}}}, state}
+
+        {:error, reason} when reason in [:replay_fail, :replay_old, :auth_fail] ->
```

Comment:
I wonder about keeping the `:auth_fail` here - If we receive such a packet it means someone has found an opened port and sent something there. Not terminating such a connection makes us vulnerable to DoS attacks. Are there any other reasons not to raise in such a case?

Reply:
Not any good ones. Let's just raise here.